config/samples/resources/computeforwardingrule/forwarding-rule-vpc-psc/compute_v1beta1_computeforwardingrule.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeForwardingRule
+metadata:
+  name: computeforwardingrule-dep-psc
+spec:
+  location: "us-central1"
+  networkRef:
+    name: computeforwardingrule-dep-psc-producer
+  subnetworkRef:
+    name: computeforwardingrule-dep1-psc-producer
+  description: "A test forwarding rule with internal load balancing scheme"
+  loadBalancingScheme: "INTERNAL"
+  backendServiceRef:
+    name: computeforwardingrule-dep-psc
+  allPorts: true
+---
+apiVersion: compute.cnrm.cloud.google.com/v1beta1
+kind: ComputeForwardingRule
+metadata:
+  name: computeforwardingrule-sample-psc
+spec:
+  description: "A VPC private service connect forwarding rule"
+  target:
+    serviceAttachmentRef:
+      name: computeforwardingrule-dep-psc
+  location: us-central1
+  networkRef:
+    name: computeforwardingrule-dep-psc-consumer
+  # PSC forwarding rule requires loadBalancingScheme to be set to empty
+  loadBalancingScheme: ""
+  allowPscGlobalAccess: true
+  ipAddress:
+    addressRef:
+      # Replace ${PROJECT_ID?} with your project ID
+      # PSC forwarding rule requires address's self_link instead of address
```

Comment:
Nit: `selfLink`; `self_link` is neither KCC's field name nor the API's field name.